Automattic/wc-calypso-bridge
# Add text domain linter

### Changes proposed in this Pull Request:



Relates to https://github.com/Automattic/i18n-issues/issues/594 and https://github.com/Automattic/wc-calypso-bridge/pull/1127



This PR is a follow-up to #1127. Users will now detect linting errors prior to committing the code to the repo. Catching linting errors on the local machine provides immediate feedback, thus leading to faster development cycles.




### How to test the changes in this Pull Request:




1. Checkout the repo and create a new branch (e.g. git checkout -b <new branch>)
2. Edit [line 32](https://github.com/Automattic/wc-calypso-bridge/blob/9f08796b234c81913749e97ce03e563182ed9c0c/includes/tasks/class-wc-calypso-task-add-domain.php#L32) of `includes/tasks/class-wc-calypso-task-add-domain.php`, and change the text domain to 'jetpack'
3. Attempt to commit the change (`git commit -a`)
4. Verify that an error occurs
5. Undo the change



### Other information:




### FOR PR REVIEWER ONLY:
